x11-plugins/gkrellm-leds/files/gkrellm-leds-0.8.2-r2-ldflags-typo.patch

```diff
@@ -0,0 +1,12 @@
+src/Makefile.am: fix type GK_FLAGS -> GKM_FLAGS
+--- a/src/Makefile.am
++++ b/src/Makefile.am
+@@ -10,7 +10,7 @@ gkleds_la_CFLAGS = -Wall -O2 -I../pixmaps @X_CFLAGS@ @GTK_CFLAGS@ @GKM_CFLAGS@
+ gkleds_la_LIBADD = @X_PRE_LIBS@ @X_LIBS@ @X_EXTRA_LIBS@ @GTK_LIBS@ @GKM_EXTRA_SRCS@
+ 
+ ## linker flags, we are building a plugin so skip lib prefix and lib versions
+-gkleds_la_LDFLAGS = -module -avoid-version @GK_LDFLAGS@
++gkleds_la_LDFLAGS = -module -avoid-version @GKM_LDFLAGS@
```

Comment:
in general, in variables you shouldn't reference `@VAR@` but `$(VAR)` because `AC_SUBST` already creates a separate `VAR = @VAR@` somewhere in `Makefile.in`.

Reply:
OK, I wanted to go for the smallest change that would fix the problem. Do you want me make this change?

Reply:
no, it's just something to be aware of if you're fixing build systems

Reply:
Alright, thanks! :smile: I appreciate it.